cumulus_library/template_sql/ctas_from_parquet.sql.jinja

```diff
@@ -2,10 +2,12 @@
 {%- if db_type == 'athena' -%}
 CREATE EXTERNAL TABLE IF NOT EXISTS `{{ schema_name }}`.`{{ table_name }}` (
 {%-  elif db_type == 'duckdb' -%}
-CREATE TABLE IF NOT EXISTS {{ table_name }} AS SELECT  
+CREATE TABLE IF NOT EXISTS "{{ table_name }}" AS SELECT  
 {%- endif %}
 {%- for col in table_cols %}
-    {{ col }}{% if db_type == 'athena' %} {{ remote_table_cols_types[loop.index0] }}{%- endif -%}
+    {%- if db_type == 'athena' %} {{ col }} {{ remote_table_cols_types[loop.index0] }}
+    {%- elif db_type == 'duckdb' %} "{{ col }}"
```

Comment:
Why only quote for duckdb and not all the time?

Reply:
Athena didn't like that, irritatingly - the serde style queries follow a slightly different set of syntax rules, it seems?